Makefile

```diff
@@ -1,5 +1,5 @@
 
-BUILD_SUBDIRS := lib billingalert awsconfig cloudtrail alarmalert volume
```

Comment:
Can you get the latest of develop branch? It looks like the addition of 'volume' is missing in your branch